### ISSUE TITLE
‘Number of Errors’ Alert in Critical state for over a year

### ISSUE THREAD

**Issue by kpujjigit** [getsentry/sentry#76580]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

It appears this Number of Errors Alert

![Image](https://github.com/user-attachments/assets/a24f8b93-52c3-40d4-a175-6ef2622a40f9)

![Image](https://github.com/user-attachments/assets/19984888-2cb9-4155-89f9-cdd89b853147)

### Expected Result

State of Alert should change, and we in-fact see a change in state when we go to Edit the rule:

![Image](https://github.com/user-attachments/assets/751f55a0-f8ad-4bcd-8644-ecbc61f73ad6)

### Actual Result

Alert has been in a critical state since it was created a year ago; insight was lost for a client and spikes occurred without alerting to proper channels.

### Product Area

Alerts

**Comment by schew2381** [comment]
@kpujjigit Hey there, could you link to the org and alert links somewhere? I don't see them on the issue rn

**Comment by leedongwei** [comment]
@kpujjigit We are still missing the link to org/alert for us to debug this.

**Comment by schew2381** [comment]
I was able to find the alert with a query. From looking at logs, the alert itself was setup in a way that it could never resolve.

The resolve threshold is set to resolve when # of errors is 100% lower than 1 week ago, but this means the number of errors at the current moment must essentially reach 0 in order resolve. 

For example, we would reach this number very close to 100% if 1 week ago we had 9991 errors, and in the current moment we have 1

![Image](https://github.com/user-attachments/assets/255e18fb-b029-4fa2-8cbe-336398191a61)

The alert was resolved by editing it, but it still has the same resolve issue so once it triggers it will never resolve as well.
